untrusted/urts/src/lib.rs

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+use mc_sgx_urts_sys::{
+    sgx_create_enclave_from_buffer_ex, sgx_destroy_enclave, sgx_enclave_id_t, sgx_status_t,
+};
+use std::ops::Deref;
+use std::{os::raw::c_int, ptr};
+
+#[derive(PartialEq, Debug)]
+pub enum Error {
+    // An error provided from the SGX SDK
+    SgxStatus(sgx_status_t),
+}
+
+/// Struct for interfacing with the SGX SDK.  This should be used directly in
+/// sgx calls `ecall_some_function(*enclave, ...)`.
+///
+/// Avoid storing the de-referenced instance of the enclave.  The de-referenced
+/// value of the enclave will result in failures to the SGX SDK after the
+/// enclave is dropped.
+#[derive(Debug, PartialEq)]
+pub struct Enclave {
+    // The enclave ID, assigned by the SGX interface
+    id: sgx_enclave_id_t,
+}
+
+/// Build an [Enclave] for use with SGX calls.
+#[derive(Default)]
```

Comment:
I don't think we need a `Default` for this builder.

Reply:
Good call!, removed